etna/transforms/math/apply_lambda.py

```diff
@@ -131,3 +133,7 @@ def inverse_transform(self, df: pd.DataFrame) -> pd.DataFrame:
                     transformed_features = self.inverse_transform_func(features)
                     result_df.loc[:, pd.IndexSlice[segments, quantile_column_nm]] = transformed_features
         return result_df
+
+    def get_regressors_info(self) -> List[str]:
+        """Return the list with regressors created by the transform."""
+        return []
```

Comment:
Here should be the same logic as [here](https://github.com/tinkoff-ai/etna/blob/645c71a5290d2cd9de7a8d12a6450ab44ae8b86c/etna/transforms/encoders/categorical.py#L156). 
If the input column is regressor, then the output column is also regressor.